homepage/homepage/app/(docs)/docs/[framework]/[[...slug]]/authentication/overview.mdx

```diff
@@ -6,12 +6,65 @@ import { CodeGroup, ContentByFramework } from "@/components/forMdx";
 
 Jazz authentication is based on cryptographic keys ("Account keys"). Their public part represents a user's identity, their secret part lets you act as that user.
 
-When a user loads a Jazz application for the first time, we create a new Account by generating keys and storing them locally.
+## Authentication States
 
-Without any additional steps the user can use Jazz normally, but they would be limited to use on only one device.
+Jazz has two distinct states for users without traditional sign-in: **Anonymous Authentication** and **Guest Mode**.
+
+### Anonymous Authentication
+
+When a user loads a Jazz application for the first time, we create a new Account by generating keys and storing them locally. This is **Anonymous Authentication**:
+
+- Users have full accounts with unique IDs
+- Data persists between sessions on the same device
+- Can be upgraded to a full account (passkey, passphrase, etc.)
+- Data syncs across the network (if enabled)
+
+### Guest Mode
+
+**Guest Mode** provides a completely accountless context:
+
+- No persistent identity or account
+- Only provides access to publicly readable content
+- Cannot save or sync user-specific data
+- Suitable for read-only access to public resources
+
+Without additional authentication steps beyond Anonymous Authentication, users can use Jazz normally but would be limited to using the application on only one device.
 
 To make Accounts work across devices, you can store/retrieve the account keys from an authentication method by using the corresponding hooks and providers.
 
+## Detecting Authentication State
+
+<ContentByFramework framework="react">
+You can detect the current authentication state using the following hooks:
+
+<CodeGroup>
+```tsx twoslash
+import * as React from "react";
+// ---cut---
+import { useAccount, useIsAuthenticated } from "jazz-react";
+
+function AuthStateIndicator() {
+  const { me } = useAccount();
+  const isAuthenticated = useIsAuthenticated();
+
+  // Check if guest mode is enabled in JazzProvider
+  const isGuest = !me && !isAuthenticated;
```

Comment:
In the GuestMode useAccount throws.

`useAccountOrGuest` must be used.

With `useAccountOrGuest` the check to detect the guest mode would be:

```ts
const isGuest = me._type !== "Account"
```

Reply:
Fixed, thanks